specs/Standards/FungibleToken/Minting.md

```diff
@@ -0,0 +1,44 @@
+# Fungible Token Minting
+
+Version `1.0.0`
+
+## Summary
+[summary]: #summary
+
+An interface for a fungible token minting, which is usually a part of a token life cycle,
+
+## Motivation
+
+Token minting can be triggered by other smart contracts. Hence we need a standard interface for minting.
+
+In a minting life cycle, token will be firstly minted and then transferred. Following scenarios are possible:
+* user can mint tokens for himself (faucet)
+* issuer will firstly mint tokens and then transfer
+* smart contract will mint tokens for a user
```

Comment:
What about adding an extra method `ft_mint_call` (similar to `ft_transfer_call`). It can be an overkill, but it could simplify some scenarios where you need to mint and immediately react to these "transfers", potentially even burning part of the tokens that are not used on a callback.

I don't have any particular use case in mind.

Reply:
Hmm, like minting for the smart contract? I think it will add more complexity, but if people think it's needed, then we can add it. We can always add it later to evaluate the need.